lib/galaxy/config.py

```diff
@@ -276,6 +276,11 @@ def __init__( self, **kwargs ):
         # On can mildly speed up Galaxy startup time by disabling index of help,
         # not needed on production systems but useful if running many functional tests.
         self.index_tool_help = string_as_bool( kwargs.get( "index_tool_help", True ) )
+        self.tool_name_boost = kwargs.get( "tool_name_boost", None )
```

Comment:
I would consider depending on config to parse a configuration file and merge some environment variables a very different dependency than depending on it to supply default values that directly relate to the business logic of the component under consideration.